style.css

```diff
@@ -0,0 +1,635 @@
+}
+
+.hackathons h3,
+p {
+  font-weight: 700;
+}
+
+.hackathon1 h3 {
+  color: #fff;
+}
+
+.hackathon1 p {
+  color: #fff;
+}
+
+.hackathon2 h3 {
+  color: #fff;
+}
+
+.hackathon2 p {
+  color: #fff;
+}
+
+.hackathon1 {
+  position: relative;
+  width: 100%;
+  max-width: 400px;
+  margin: 20px 5px;
+  padding: 50px 30px;
+  background-color: rgba(255, 68, 0, 0.507);
+}
+
+.hackathon1::before {
+  content: '';
+  width: 100%;
+  max-width: 400px;
+  position: absolute;
+  background-image: url('./images/hackjos1.jpg');
+  background-repeat: no-repeat;
+  background-size: cover;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  z-index: -1;
+  opacity: 0.4;
+}
+
+.hackathon2 {
+  position: relative;
+  width: 100%;
+  max-width: 400px;
+  margin: 20px 5px;
+  padding: 50px 30px;
+  background-color: rgba(255, 68, 0, 0.507);
+}
+
+.hackathon2::before {
+  content: '';
+  width: 100%;
+  max-width: 400px;
+  position: absolute;
+  background-image: url('./images/hackjos2.jpg');
+  background-repeat: no-repeat;
+  background-size: cover;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  z-index: -1;
+  opacity: 0.4;
+}
+
+.featured-speakers .section-heading {
+  color: #272a31;
+}
+
+.featured-speakers-container {
+  display: grid;
+  grid-template-columns: auto auto;
+  grid-auto-flow: row;
+  gap: 25px;
+}
+
+.all-speakers-container {
+  display: grid;
+  grid-template-columns: auto auto;
+  grid-auto-flow: row;
+  gap: 25px;
+}
+
+.featured-speakers-card {
+  position: relative;
+  max-width: 700px;
+  display: flex;
+}
+
+.featured-speakers-card::after {
+  content: '';
+  position: absolute;
+  background-image: url('./images/black-white2.jpg');
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 70px;
+  height: 70px;
+  top: 0;
+  left: 0;
+  opacity: 0.6;
+  z-index: -1;
+}
+
+.speaker-img {
+  width: 200px;
+  max-width: 180px;
+  max-height: 130px;
+  padding-left: 10px;
+  padding-top: 10px;
+}
+
+.speaker-img img {
+  width: 100%;
+  height: 100%;
+}
+
+.featured-text {
+  padding-left: 20px;
+  max-width: 400px;
+}
+
+.featured-text i {
+  color: #ec5242;
+}
+
+.featured-text hr {
+  width: 30px;
+}
+
+.get-more {
+  display: none;
+  border: 1px solid #d3d3d3;
+  padding: 20px;
+  align-items: center;
+}
+
+.more {
+  margin: 0 auto;
+  border: none;
+  font-size: 20px;
+  background-color: inherit;
+}
+
+.fa-angle-down {
+  color: #ec5242;
+  padding-left: 5px;
+}
+
+.partner-section {
+  position: relative;
+  margin-top: 40px;
+  background-color: #272a31;
+  z-index: 1;
+  padding-bottom: 50px;
+}
+
+.partners-link {
+  justify-content: center;
+  flex-wrap: wrap;
+  max-width: 700px;
+  margin: 0 auto;
+}
+
+.partners-link img {
+  width: 100%;
+  height: 80px;
+  filter: grayscale(90%);
+}
+
+.footer-section {
+  position: relative;
+  margin-top: 30px;
+  display: grid;
+  grid-template-columns: auto auto;
+  gap: 35px;
+  margin-bottom: 20px;
+  align-items: center;
+}
+
+@media (max-width: 768px) {
```

Comment:
- [ ] You are supposed to follow a mobile-first approach. That means the default styling is for the mobile view and you will have media query for width 768 and beyond. Kindly make your styling mobile first. :pray:

Reply:
- [ ] Here I'll try to be more specific. Talking about the default style could be for mobiles is something like this:

```
/* This elements (outside a media parenthesis) are the default */
.head {
 some-element: 3px;
}

@media (min-width: 768px) {
/* All this is for tablet and desktop size */
}
```

And actually this is what you've:

```
/* This elements (outside a media parenthesis) are the default */
.head {
 some-element: 3px;
}

@media (min-width: 768px) {
/* All this is for tablet and desktop size */
}

@media (max-width: 768px) {
/* All inside here (the mobile style), could be at the top without any @media option */
}
```


And that's it, I see on your style you're using the default, 768 px or less and 768 or more, talking about 768px or less is not necessary for a mobile-first design because this could be at the top of the script. Please, make the required changes to have your project according to a mobile-first. 🌵